widget_driver_test/README.md

```diff
@@ -57,6 +57,32 @@ await tester.pumpWidget(myWidget);
 // Do your widget testing now
 ```
 
+### Use real drivers in your DrivableWidgets
+
+In case you have some special use case where you actually don't want to use `TestDrivers` then you can use the `WidgetDriverTestConfigProvider`. It gives you the possibility to control if a TestDriver or a real driver is created for each `DrivableWidget`. Just wrap the widget under tests inside a `WidgetDriverTestConfigProvider`.
```

Comment:
I would add a note that usually this is a bad idea and kills one of the benefits you get from using WD.

Reply:
Good point @dominik089 . I just added a small `NOTE` about this in the README.